include/metaverse/explorer/extensions/commands/addnode.hpp

```diff
@@ -73,7 +73,17 @@ class addnode: public command_extension
             "ACCOUNTAUTH",
             value<std::string>(&auth_.auth)->required(),
             BX_ACCOUNT_AUTH
-	    );
+	    )
+        (
+            "address,a",
```

Comment:
address parameter should be required, and better be positional argument.

Reply:
OK, change to positional argument:
Usage: mvs-cli addnode [-h] ACCOUNTNAME ACCOUNTAUTH NODEADDRESS          

Info: addnode                                                            

Options (named):

-h [--help]          Get a description and instructions for this command.

Arguments (positional):

ACCOUNTNAME          Account name required.                              
ACCOUNTAUTH          Account password(authorization) required.           
NODEADDRESS          The target node address[x.x.x.x:port].